docs/quickstart.rst

```diff
@@ -83,6 +85,56 @@ And using the ``:class:`` parameter in combination with custom css, you can chan
 .. video:: _static/video.mp4
     :class: video-bordered
 
+Alignment: 
+
+.. code-block:: rst
+
+    .. video:: _static/video.mp4
+        :align: left
+
+.. video:: _static/video.mp4
+    :align: left
+
+.. code-block:: rst
+
+    .. video:: _static/video.mp4
+        :align: center
+
+.. video:: _static/video.mp4
+    :align: center
+
+.. code-block:: rst
+
+    .. video:: _static/video.mp4
+        :align: right
+
+.. video:: _static/video.mp4
+    :align: right
+
+For consistency with previous versions, which not support align, the default value of align 'left' when the it is not set.
+If you want to use the 'default' defined by theme, you need to, manually, set it to 'default':
```

Comment:
default: <not setted>  = left
however, setting to 'default' = <the default in theme>
The puzzling word 'default'.

Or no longer required to be consistent with the previous versions?

Reply:
sphinx_book_theme, pydata_sphinx_theme

Reply:
Please decide default alignment: follow past versions, or let the theme control?
 Once determined and modified you can merge.

Reply:
The one you coded fits very well with the past version let's keep it